Mathlib/CategoryTheory/Enriched/Limits/IsConicalLimit.lean

```diff
@@ -0,0 +1,152 @@
+/-
+Copyright (c) 2025 Jon Eugster. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Dagur Asgeirsson, Jon Eugster, Emily Riehl
+-/
+import Mathlib.CategoryTheory.Enriched.Ordinary.Basic
+import Mathlib.CategoryTheory.Limits.Preserves.Limits
+
+/-!
+# Conical limits in enriched ordinary categories
+
+A limit cone in the underlying category of an enriched ordinary category is a *conical limit* if
+it is a limit cone in the underlying ordinary category and moreover this limit cone is preserved
+by covariant enriched representable functors. These conditions are encoded in the structure
+`IsConicalLimit`.
+
+The universal property of a conical limit is enriched as follows: the canonical comparison map
+defines an isomorphism in the enriching category:
+
+`limitComparisonIso (h : IsConicalLimit V c) : (X ⟶[V] c.pt) ≅ (limit (F ⋙ eCoyoneda V X))`
+
+Conversely, if the canonical maps define isomorphisms for all `X` then `c` is a conical limit cone:
+
+`ofIsIsoLimitComparison [∀ X, IsIso (IsConicalLimit.limitComparison V c X)]`
+`(hc : IsLimit c) : IsConicalLimit V c`
+
+This file develops some general API for conical limits in enriched ordinary categories.
+
+TODO: Dualize everything to define conical colimits.
+-/
+
+universe v₁ u₁ w v' v u u'
+
+namespace CategoryTheory.Enriched
+
+open Limits
+
+variable {J : Type u₁} [Category.{v₁} J]
+variable (V : Type u') [Category.{v'} V] [MonoidalCategory V]
+variable {C : Type u} [Category.{v} C] [EnrichedOrdinaryCategory V C]
+variable {F : J ⥤ C} (c : Cone F) (X : C)
+
+/--
+A limit cone `c` in a `V`-enriched ordinary category `C` is a *`V`-enriched limit*
+(or *conical limit*) if for every `X : C`, the cone obtained by applying the coyoneda
+functor `(X ⟶[V] -)` to `c` is a limit cone in `V`.
+-/
+structure IsConicalLimit where
+  /-- A conical limit cone is a limit cone. -/
+  isLimit : IsLimit c
+  /--
+  The cone obtained by applying the coyoneda functor `(X ⟶[V] -)` to `c` is a limit cone in `V`.
+  -/
+  isConicalLimit (X : C) : IsLimit <| (eCoyoneda V X).mapCone c
+
+namespace IsConicalLimit
+
+variable {V} {c}
+
+/-- Transport evidence that a cone is a `V`-enriched limit cone across an isomorphism of cones. -/
+noncomputable def ofIso {r₁ r₂ : Cone F} (h : IsConicalLimit V r₁) (i : r₁ ≅ r₂) :
+    IsConicalLimit V r₂ where
+  isLimit := h.isLimit.ofIsoLimit i
+  isConicalLimit X := h.isConicalLimit X |>.ofIsoLimit
+    { hom := Functor.mapConeMorphism _ i.hom
```

Comment:
This suggests there is a missing API: `Functor.mapConeIso` in `CategoryTheory.Limits.Cones`.